app/components/icons/FilterIcon.tsx

```diff
@@ -1,6 +1,6 @@
 export default function FilterIcon({ classNames = "h-6 w-6" }) {
   return (
-    <div style={{marginRight:"1rem"}}>
+    <div style={{marginRight:"1rem"}} className="filter-icon">
```

Comment:
You can use a Tailwind class instead of the `style` tag here – `mr-4` should result in the same outcome. Using `style` here does two things you most of the time do not want in a project:

It introduces a different way of styling next to what is generally used (Tailwind in this case).

The other thing is the importance of styles – a directly applied style has precedence over any `class(Name)` added to an element. In our setup, that most certainly won't clash with anything, but in bigger projects, you could easily introduce weird bugs through this. So best try to stick to the Tailwind classes wherever possible :)